docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '2'
+services:
+    app:
+      image: node:6.11
+      working_dir: /usr/src/app
+      ports:
+        - "9090:9090"
+        - "7357:7357"
+        - "7358:7358"
+      volumes:
+          - ./src/gd-frontend:/usr/src/app
```

Comment:
Minor nitpick, but the indentation levels in the volumes block don't match the rest of the file. This is also the case in docker-compose.test.yml